src/lib.rs

```diff
@@ -12,10 +12,14 @@ pub use axum_tracing_opentelemetry::opentelemetry_tracing_layer;
 
 pub mod propagation;
 
-pub fn init_tracing(log_level: tracing::Level) {
+pub fn init_tracing_with_fallbacks(
+    log_level: tracing::Level,
+    service_name_fallback: &'static str,
+    service_version_fallback: &'static str,
+) {
     let otel_rsrc = DetectResource::default()
-        .with_fallback_service_name(env!("CARGO_PKG_NAME"))
-        .with_fallback_service_version(env!("CARGO_PKG_VERSION"))
+        .with_fallback_service_name(service_name_fallback)
```

Comment:
can we keep the variables name with the same pattern as the function? for example `fallback_service_name`

Reply:
https://github.com/nentgroup/telemetry-rust/pull/1/commits/c1b187577056727890b0bc73174fae9c7eb3cb60